src/api/hooks/useGetANS.ts

```diff
@@ -45,6 +55,8 @@ export const knownAddresses: Record<string, string> = {
     "LiquidSwap v0.5",
   "0x54cb0bb2c18564b86e34539b9f89cfe1186e39d89fce54e1cd007b8e61673a85":
     "LiquidSwap v1",
+  "0xb247ddeee87e848315caf9a33b8e4c71ac53db888cb88143d62d2370cca0ead2":
+    "LiquidSwap v1 Farms",
```

Comment:
can you also add Thala farm "0x6b3720cd988adeaf721ed9d4730da4324d52364871a68eac62b46d21e4d2fa99"